projects/canopy/src/lib/carousel/auto-play/auto-play.component.html

```diff
@@ -5,7 +5,13 @@
   type="button"
 >
   <span class="lg-visually-hidden">Stop Animation</span>
-  <lg-icon name="pause"></lg-icon>
+  <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+    viewBox="0 0 500 500" style="enable-background:new 0 0 500 500;" xml:space="preserve">
+  <g>
+    <path class="carousel-icon" d="M250.09,1C112.63,1,1.2,112.43,1.2,249.89s111.43,248.89,248.89,248.89s248.89-111.43,248.89-248.89
+      S387.55,1,250.09,1z M231.82,353.79h-77.64V146h77.64V353.79z M346,353.79h-77.64V146H346V353.79z"/>
+  </g>
+  </svg>
```

Comment:
I'm a bit weary of replacing `lg-icon` with an SVG directly in the file. Ideally we'd use the icon component so that there's always consistency across all icons in the app. Is there a problem with the original SVG file for these icons? If so, I think it would be better to edit/update those instead (while preserving the ability to change the colour). I've had issues before where icons weren't centered inside the SVG canvas properly, so may be something like that? Otherwise, perhaps you could align them by nudging a container? If there's no other way I'm happy with this change, but it's worth checking.

Reply:
Yes, I agree with what @owensgit said. Could you point out what styles were problematic in the `lg-icon`? we might want to update the `lg-icon` itself 🤔

Reply:
The problem is the svg was just for the play / pause icons and does not include the circle shape (which was handled by the button) These svgs include all shapes needed for the buttons

Reply:
I could add them to the list of lg-icons as **carousel-play** and **carousel-pause**

Reply:
I think adding new icons rather than editing existing ones is a better way to go

Reply:
Yes, I was looking at the existing icons and I only found the play one with the circle but even that would be wrong since the actual play would take the colour of the background e.g. 

![Screenshot 2022-05-17 at 10 58 23](https://user-images.githubusercontent.com/8397116/168785204-a7ea1646-ccc7-4735-b08a-fdf83240552b.png)

I'm not sure I see these icons used elsewhere so I'm wondering if we should just keep them in here? @owensgit what do you think? 
Usually for videos we keep the play buttons of the players.

Reply:
I'd be happy to keep the SVG code in this component, as a one off for the carousel, considering those icons won't be used anywhere else 👍

Reply:
@dannyoz I just deployed the branch and noticed that the play/pause icons take the background colour so I'd say you can use the `play-spot` icon and then ask one of the designers to create a new icon `pause-spot`. They will need to add it to the design library too.